src/providers/image-loader.ts

```diff
@@ -78,16 +78,25 @@ export class ImageLoader {
     private fileTransfer: FileTransfer,
     private platform: Platform
   ) {
-    Observable.fromEvent(document, 'deviceready').first().subscribe(res => {
-      if (this.nativeAvailable) {
-        this.initCache();
-      } else {
+    platform.ready().then(() => {
+      if (!platform.is('cordova')) {
         // we are running on a browser, or using livereload
         // plugin will not function in this case
-        this.isInit = true;
+        this.init = true;
```

Comment:
Why did you change this `isInit` to `init` but the other instances still read `isInit`